src/test/datascience/debugger.vscode.test.ts

```diff
@@ -118,7 +118,7 @@ suite('VSCode Notebook - Run By Line', function () {
             );
 
             // Give the files a quick chance to clean up
-            await sleep(1000);
+            await sleep(3000);
```

Comment:
Is this necessary?

Reply:
Test was failing attempting to see if files were cleaned up. Stopped failing after I did this, so yeah probably.

There's likely another way to accomplish this but I wasn't working on this code so I just wanted the test to pass

Reply:
Ok that makes sense! Thanks for the context